kairon/shared/pyscript/callback_pyscript_utils.py

```
@@ -335,4 +335,14 @@ def save_as_pdf(text: str, bot: str, sender_id:str):
             )
             return media_id
         except Exception as e:
-            raise Exception(f"encryption failed-{str(e)}")
\ No newline at end of file
+            raise Exception(f"encryption failed-{str(e)}")
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)